sdk/iot/Azure.Iot.Hub.Service/src/Authentication/ISasTokenProvider.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.Hub.Service.Authentication
+{
+    /// <summary>
+    /// The token provider interface for shared access signature based authentication.
+    /// </summary>
+    public interface ISasTokenProvider
```

Comment:
This should be internal.

Reply:
Good point; I thought I made all of these internal, I must have missed it. I'll check all the auth classes